torch/_tensor.py

```diff
@@ -1012,6 +1013,61 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             else:
                 return _convert(ret, cls)
 
+    def __dlpack__(self, stream=None):
+        """
+        Creates a dlpack capsule of the current tensor to
+        be exported to other libraries.
+
+        This function will be called from the `from_dlpack` method
+        of the consumer library.
+
+        Args:
+            stream (integer or None): A Python integer representing a pointer
+            to a stream (CUDA or ROCm). `stream` is provided by the consumer
```

Comment:
Is there a place we can check the rendered doc? I thought from the 2nd line onward we need an indentation?

Reply:
There normally is, in the CI list there's a doc artifact link. However, on this run the doc build failed with an unrelated issue:
```
/var/lib/jenkins/workspace/docs/source/notes/cuda.rst:270: WARNING: Definition list ends without a blank line; unexpected unindent.
```
so no artifact was generated.

Reply:
Rendered docs are now available as a link on the Dr. CI post:

https://pytorch.org/docs/master/dlpack.html?highlight=dlpack#torch.utils.dlpack.to_dlpack